File-Formats/DocIO/Open-and-save-Word-document-in-MAUI.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Open and save Word document in .NET MAUI | Syncfusion
+description: Open and save Word document in .NET MAUI application using Syncfusion .NET MAUI Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Open and Save Word document in .NET MAUI
+
+Syncfusion DocIO is a [.NET MAUI Word library](https://www.syncfusion.com/document-processing/word-framework/maui/word-library) used to **create, read, and edit Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **open and save a Word document in .NET MAUI**.
+
+**Prerequisites:**
+To create .NET Multi-platform App UI (.NET MAUI) apps, you need the latest versions of Visual Studio 2022 and .NET 6. For more details, refer [here](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-7.0&tabs=vswin).
+
+## Steps to open and save Word document programmatically in .NET MAUI
+
+Step 1: Create a new C# .NET MAUI app. Select **.NET MAUI App (Preview)** from the template and click the **Next** button.
+
+Step 2: Enter the project name and click **Create**.
+
+Step 3: Install the [Syncfusion.DocIO.NET](https://www.nuget.org/packages/Syncfusion.DocIO.Net) NuGet package as a reference to your project from [NuGet.org](https://www.nuget.org/).
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering a Syncfusion license key in your application to use our components.
+
+Step 4: Add a new button to the **MainPage.xaml** as shown below.
+
+{% tabs %}
+
+{% highlight XML %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```

Comment:
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
            xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
            x:Class="OpenAndSave.MainPage"
            BackgroundColor="{DynamicResource SecondaryColor}">
    <ScrollView>
        <Grid RowSpacing="25" RowDefinitions="Auto,Auto,Auto,Auto,*"
            Padding="{OnPlatform iOS='30,60,30,30', Default='30'}">
            <Button 
                Text="Open and save"
                FontAttributes="Bold"
                Grid.Row="0"
                SemanticProperties.Hint="Creates Word document you click"
                Clicked="OpenAndSaveDocument"
                HorizontalOptions="Center" />
        </Grid>
    </ScrollView>
</ContentPage>